modules/release-notes/ref-release-notes-fixed-issues.adoc

```diff
@@ -2,8 +2,79 @@
 [id="fixed-issues"]
 = Fixed issues
 
-This section lists issues fixed in {product} {product-version}.
+This section lists issues fixed in {product} {product-version} that have a significant impact on users.
```

Comment:
the suffix "that have a significant impact on users." was manually removed in the docs for 1.5, but the release notes scripted template was NOT updated (so it's re-appearing here). 

See PRs to remove the same thing for 1.6+ 
* https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/1143 (1.6)
* https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/1145 (1.5)
* https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/1146 (main)